src/axe-tool-property-provider.test.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { getAxeToolProperties } from './axe-tool-property-provider';
+import * as Sarif from './sarif/sarif-2.0.0';
+
+describe('axe-tool-properties', () => {
+    describe('getAxeToolProperties', () => {
+        it("returns the axe properties as a Sarif.Run['tool']", () => {
+            const axeToolPropertiesStub: Sarif.Run['tool'] = {
```

Comment:
>axeToolPropertiesStub [](start = 18, length = 21)

nit: this isn't a stub; this is a real, valid result. `expectedResults` would be a better name for this, I think.